namecomplete/candy.js

```diff
@@ -81,15 +81,7 @@ CandyShop.NameComplete = (function(self, Candy, $) {
 
 				});
 
-				// if we only have one match, no need to show the picker, just replace it
-				// else show the picker of the name matches
-				if (matches.length === 1) {
-					self.replaceName(matches[0]);
-					// Since the name will be autocompleted, throw away the last character
-					e.preventDefault();
-				} else if (matches.length > 1) {
-					self.showPicker(matches, field);
-				}
+				self.showPicker(matches, field);
```

Comment:
another thing, even in english, this behavior is still not suitable. 
It will cause such problem
1. when I don't want mention. for example i just want to type `@zh` like previous post, it will autocomplete `@zh99998: ` without confirm.
2. when I type fast without looking at it, I input `@zh99998`, it will be `@zh99998: 99998`.